package.json

```diff
@@ -60,7 +60,7 @@
 		"packages/*"
 	],
 	"lint-staged": {
-		"**/*": "prettier --write --ignore-unknown --allow-empty --loglevel debug"
+		"**/!(compile_commands.json)*": "prettier --write --ignore-unknown --allow-empty --log-level debug"
```

Comment:
Could this be added in `.prettierignore`?

Reply:
You would think so but nope! Prettier doesn't have a problem when run on its own, the problem comes from `lint-staged`. `lint-staged` will use that pattern to find files but resolves the files to actual paths that it gives to prettier and prettier will, if explicitly given a path to a symlink, error out.

Reply:
yikes